package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Dependent-Jaguar, MetroRail: Data Modeling & Database Design (LG Goal 126)",
   "main": "app.js",
   "scripts": {
-    "test": "mocha --watch dist/spec",
+    "test": "./node_modules/mocha/bin/_mocha",
```

Comment:
Do you now have to run the `mocha` executable directly from the `node_modules` folder?

Might want to look into [why](http://stackoverflow.com/questions/9722407/how-do-you-install-and-run-mocha-the-node-js-testing-module-getting-mocha-co) if you have time.